platform/nodejs/main.js

```diff
@@ -101,6 +101,21 @@ async function enableWASM() {
 function pslInit(raw) {
     if ( typeof raw !== 'string' || raw.trim() === '' ) {
         const require = createRequire(import.meta.url); // jshint ignore:line
+
+        let serialized = null;
+
+        // Use serialized version if available
+        try {
+            serialized = require('./build/publicsuffixlist.json');
+        } catch (error) {
+        }
+
+        if (serialized !== null) {
+            globals.publicSuffixList.parse('//', globals.punycode.toASCII);
```

Comment:
@gorhill is this line even necessary?

Reply:
No not necessary.

Reply:
I've updated the patch now.

There was also a call to `JSON.parse()` that was breaking the implementation. `require()` already parses the JSON.